channels/disp/client/disp_main.c

```diff
@@ -79,40 +80,40 @@ typedef struct _DISP_PLUGIN DISP_PLUGIN;
  *
  * @return 0 on success, otherwise a Win32 error code
  */
-UINT disp_send_display_control_monitor_layout_pdu(DISP_CHANNEL_CALLBACK* callback, UINT32 NumMonitors, DISPLAY_CONTROL_MONITOR_LAYOUT* Monitors)
+UINT disp_send_display_control_monitor_layout_pdu(DISP_CHANNEL_CALLBACK* callback,
+        UINT32 NumMonitors, DISPLAY_CONTROL_MONITOR_LAYOUT* Monitors)
 {
 	UINT status;
 	wStream* s;
-	UINT32 type;
 	UINT32 index;
-	UINT32 length;
 	DISP_PLUGIN* disp;
 	UINT32 MonitorLayoutSize;
-
 	disp = (DISP_PLUGIN*) callback->plugin;
+	MonitorLayoutSize = DISPLAY_CONTROL_MONITOR_LAYOUT_SIZE;
+	DISPLAY_CONTROL_HEADER header =
```

Comment:
Sorry, named initializers for structs are not supported well enough :/ (looking at you, vs2010)

Reply:
Oh, and declaring variables mid code ;)